### ISSUE TITLE
Can't set axis label color with multiple values

### ISSUE THREAD

**Issue by AlFontal** [has2k1/plotnine#329]
I am trying to manually set the color of my axis labels with a list of colors of equal length as the labels, just as you can do with R.

I tried to make a minimal example to showcase the difference in behaviour between `ggplot2` and `plotnine` in this regard:

#### In `ggplot2`:
```R
colors = rep(c('blue', 'red', 'orange', 'green'), 8)

ggplot(mtcars) + 
  aes(name, wt) + 
  geom_col() + 
  coord_flip() + 
  theme(axis.text.y=element_text(color=colors))
```

![image](https://user-images.githubusercontent.com/10958332/68872064-b7e5ba80-06fd-11ea-8ce5-7b950b3ce4ed.png)

#### In `plotnine`:

```python
colors = ['blue', 'red', 'orange', 'green'] * 8

(ggplot(mtcars) 
  + aes('name', 'wt')  
  + geom_col()  
  + coord_flip()  
  + theme(axis_text_y=element_text(color=colors)))
```
```
 ValueError: Invalid RGBA argument: ('blue', 'red', 'orange', 'green', 'blue', 'red', 'orange', 'green', 'blue', 'red', 'orange', 'green', 'blue', 'red', 'orange', 'green', 'blue', 'red', 'orange', 'green', 'blue', 'red', 'orange', 'green', 'blue', 'red', 'orange', 'green', 'blue', 'red', 'orange', 'green') 
```
It looks like you can only set a single color for all the ticks but I have no clue on how to individually set the color for each of the labels.

With `matplotlib` figure objects you can expose and modify the tick labels by using `plt.gca().get_xticklabels()`. Is there any way to access to this level in a `plotnine` figure?

**Comment by has2k1** [comment]
This is a rarely used feature that I ignored because of the amount of code required to support for every attribute that you can theme. But, you can drop into Matplotlib for this.
 
```python
colors = ['blue', 'red', 'orange', 'green'] * 8
fig = (ggplot(mtcars) 
  + aes('name', 'wt')  
  + geom_col()  
  + coord_flip()
).draw()

ax = fig.axes[0]
labels = ax.get_yticklabels()

for l, c in zip(labels, colors):
    l.set_color(c)
```

I look into it again.

**Comment by AlFontal** [comment]
Indeed, after over a year of intense use of `plotnine` I never needed to use this feature and I seem to be the first to ask, so it is probably not worth including it given that we can acces the Matplotlib figure directly with `.draw()`.

Thanks!

**Comment by projectsUW** [comment]
I would like this to work, too.

**Comment by mikenlee** [comment]
@projectsUW I agree. It's a great way to subtly draw someone's eyes to a part of a visualization without having to add colors to the main part of the chart. It would be very clean to be able to do this all under 1 roof.

**Comment by AlFontal** [comment]
Almost 5 years later, I end up finding my own issue again 😆 

@has2k1 I was just wondering if there's been some change in the API to ease this process so as to not have to drop to Matplotlib to label the axis text. I know there's been quite a revamp of the theming elements so it might be possible now? 🤔 

Thanks in any case!

**Comment by has2k1** [comment]
This has been possible for a while. So this works.


```python
colors = ["blue", "red", "orange", "green"] * 8

(ggplot(mtcars) 
  + aes("name", "wt")  
  + geom_col()  
  + coord_flip()  
  + theme(axis_text_y=element_text(color=colors))
)
```
